src/App.js

```diff
@@ -37,14 +38,17 @@ const donations = [
 ];
 
 function App() {
+  const myDonationForm = DonationForm
+  console.log(donations)
   return (
     <>
       <TopBar />
       <main className="container">
-        <section className="sidebar">{/* Recent Donations */}</section>
+        <section className="sidebar">RecentDOnations donations = {donations}</section>
         <section className="">
           {/* Progress */}
-          {/* Donation Form */}
+          <DonationForm donations = {donations} />
+          <Progress donations = {donations} />
```

Comment:
The order in which you lost the components is important. The progress should be before your form.